### PR TITLE
fix: only return smart title if original title is detected as clickbait

### PR DESCRIPTION
**Pull request by omBratteng**

### CODE REVIEW

File: src/graphorm/index.ts [hunk 1]
```diff
@@ -72,6 +72,14 @@ const nullIfNotSameUser = <T>(
   return ctx.userId === user.id ? value : null;
 };
 
+const checkIfTitleIsClickbait = (value?: string): boolean => {
+  const clickbaitProbability = parseFloat(value || '0');
+  const threshold = remoteConfig.vars.clickbaitTitleProbabilityThreshold || 1;
+  console.log('clickbaitProbability', clickbaitProbability);
```
Comment (rebelchris) [inline src/graphorm/index.ts:78]: Testing?

Comment (rebelchris) [inline src/graphorm/index.ts:78]:
```suggestion
```

**Review by rebelchris — APPROVED** [review]
Looks good.
Did we set clickbait shield by default on for plus?

File: src/graphorm/index.ts (hunk 1 above)
Comment (omBratteng) [reply src/graphorm/index.ts]: Yup

**Comment by omBratteng** [comment]
> Did we set clickbait shield by default on for plus?

Yeah

**Pull request merged**